src/commands/validate.ts

```diff
@@ -4,21 +4,29 @@ import Command from '../base';
 import { ValidationError } from '../errors/validation-error';
 import { load } from '../models/SpecificationFile';
 import { SpecificationFileNotFound } from '../errors/specification-file';
+import { specWatcher } from '../globals';
 
 export default class Validate extends Command {
   static description = 'validate asyncapi file';
 
   static flags = {
-    help: flags.help({ char: 'h' })
+    help: flags.help({ char: 'h' }),
+    watch: flags.boolean({
```

Comment:
maybe we could do it more generic, so we do not have to repeat the description of the flag everywhere. So it is basically not `flags.boolean` but `flags.watch`? @Souvikns @magicmatatjahu @boyney123 thoughts?

Reply:
@derberg  Do you mean, building a wrapper over `flags.boolean`

Reply:
@imabp Yep, but I think that Łukasz thinks about creating reusable flag and then reuse it in other commands. You can check documentation how do that in the ocliff https://oclif.io/docs/flags#custom-flags

Reply:
Sure, thank you @magicmatatjahu 
I was looking at that only.

Just figuring out, how do I pass on the `current command`,`commandName` and `filepath` to the `specWatcher.`

Reply:
Thanks @magicmatatjahu for the resource. I have refactored the watcher.  Please review @derberg @magicmatatjahu

Reply:
I am also figuring it out, how to make it more better, just like help command, need some time to explore those parts.